plugins/bash/init.lua

```
@@ -595,6 +677,38 @@ rtk_single_command = function(command)
   return not root:has_error() and not is_complex(root) and #collect_commands(root, command) == 1
 end
 
+rtk_explicit_proxy = function(command)
```

Luacheck (1.2.0)

[warning] setting non-standard global variable 'rtk_explicit_proxy'

(W111)